packages/pilot/src/pages/Account/actions/epic.js

```diff
@@ -101,10 +113,17 @@ const companyEpic = (action$, state$) =>
           return Promise.resolve(payload)
         }
 
-        return client.company.current().catch(identity)
+        return client.company.current()
+          .then((company) => {
```

Comment:
essa função pode ser definida acima visto que não usa nada alem dos parametros de entrada, apenas para facilitar a leitura desse trecho, algo como 
```suggestion
const verifyEnvironmentPermission = (company) => {
  if (env === 'live' && !isActiveCompany(company)) {
    throw new Error('Unauthorized environment')
  }

  return company
}
....
return client.company.current()
          .then(verifyEnvironmentPermission)
```